django_superform/widgets.py

```diff
@@ -47,17 +46,28 @@ def render(self, name, value, attrs=None, **kwargs):
         if template_name is None:
             template_name = self.template_name
         context = self.get_context(name, value, attrs=attrs or {}, **kwargs)
-        return loader.render_to_string(
-            template_name,
-            dictionary=context,
-            context_instance=self.context_instance)
+        return loader.render_to_string(template_name, context=context)
 
 
 class FormWidget(TemplateWidget):
     template_name = 'superform/formfield.html'
     value_context_name = 'form'
 
+    @property
+    def media(self):
+        return self.field.form_class().media
```

Comment:
Instantiating the `form_class` / `formset_class` to get the media can cause some problems. I'll try to see if there's a way around it.